pkg/operation/botanist/etcd.go

```diff
@@ -108,6 +110,13 @@ func (b *Botanist) DeployEtcd(ctx context.Context) error {
 			Container:            string(secret.Data[v1beta1constants.DataKeyBackupBucketName]),
 			FullSnapshotSchedule: snapshotSchedule,
 		})
+
+		if gardencorev1beta1helper.SeedSettingOwnerChecksEnabled(b.Seed.GetInfo().Spec.Settings) {
```

Comment:
Shouldn't we also check for the UseDNSRecords feature gate here? If it is disabled, then the owner DNS record would not exist.

Reply:
Yes, absolutely, thanks for noticing. I also thought about double-checking that the feature gate and the seed setting are always checked together (whenever owner checks are concerned) but somehow it slipped my mind.

Reply:
Done, PTAL